examples/modules/sulllivtr-graphql/main.tf

```diff
@@ -0,0 +1,24 @@
+
+provider "graphql" {
+  url = var.service.graphiql_url
+  headers = {
+    "Content-Type" = "application/json"
+    "API-Key" = var.service.api_key
+  } 
+}
+
+resource "newrelic_browser_application" "access_data" {
+  name = "New relic access data info"
+}
+
+data "graphql_query" "basic_query" {
+  query_variables = {
+    "id" = var.service.key_id
+    "key_type" = var.service.key_type
+  }
+  query  = file("query.gql")
+}
+
```

Comment:
@vinay-newrelic it might be a good idea for us to add the `newrelic_api_access_key` resource also to this module. This module would anyway be used by a customer after they create an API key via the `newrelic_api_access_key` resource (using the ID of the created API key with this module), so it would be a good idea to integrate this resource into the module too.

This would allow customers to be able to actually get the API key directly out of the `newrelic_api_access_key` resource; the only difference it would make to the functioning of this module would be that the query variable "id" would be `newrelic_api_access_key.foo.id` (the ID exported by the resource).